app/views/admin/export_domain_blocks/_domain_block.html.haml

```diff
@@ -23,5 +23,5 @@
         = f.object.public_comment
       - if existing_relationships
         ·
-        = fa_icon 'warning fw'
+        = material_symbol 'warning', class: 'fw'
```

Comment:
> I do not think the `fw` class should be kept. This is FontAwesome specific to ensure that the icon has a fixed width (= is a square), which is always the case for the material icons I think?

I'll do a pass removing usage of that class, and try to verify they are all square where we do so.

> And otherwise it should not work if the icon does not have the `fa` class.

Can you clarify that?

> The only think to look for is if we specifically target the `fw` class in our styles, but I dont think we do.

Yes, I think it's used from the supplied FontAwesome stylesheets, but not used/extended in our own styles.